src/t_hash.c

```diff
@@ -322,6 +321,24 @@ static struct listpackEx *listpackExCreate(void) {
     return lpt;
 }
 
+void *listpackExCreateFromListpack(void *lp) {
+    listpackEx *lpt = zcalloc(sizeof(*lpt));
+    lpt->lp = lp;
+    lpt->meta.trash = 1;
+    lpt->key = NULL;
+    return lpt;
+}
+
+void *listpackExGetListpack(const robj *o) {
+    listpackEx *lpt = o->ptr;
+    return lpt->lp;
+}
+
+void listpackExUpdateListpack(const robj *o, void *lp) {
+    listpackEx *lpt = o->ptr;
+    lpt->lp = lp;
+}
```

Comment:
since listpackEx can be accessed outside server.h, i think we can discard these methods, and listpackExCreateFromListpack() is only used in rdb.c, it's not a good deal to add a separate for it.